Assets/Tools/PivotAlignment/PivotAllignment_window.cs

```diff
@@ -0,0 +1,96 @@
+using UnityEngine;
+using UnityEditor;
+
+public class PivotAlignment_window : EditorWindow
+{
+    private static PivotAlignment_window win;
+    private GameObject gameObject;
+    private MeshRenderer renderer;
+    private string[] options = new string[] { "Original", "Center", "Min", "Max" };
+    private int selectedIndexX = 0;
+    private int selectedIndexY = 0;
```

Comment:
When possible, lets try to keep variable names as descriptive but brief as possible when it comes to what its used for.

I think that something like `selectedAlignmentY` is more telling of what this is. What are your thoughts?